src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1975,7 +1976,10 @@ def aks_agentpool_delete(cmd,   # pylint: disable=unused-argument
         raise CLIError("Node pool {} doesnt exist, "
                        "use 'aks nodepool list' to get current node pool list".format(nodepool_name))
 
-    return sdk_no_wait(no_wait, client.begin_delete, resource_group_name, cluster_name, nodepool_name)
+    if ignore_pod_disruption_budget is True:
+        return sdk_no_wait(no_wait, client.begin_delete, resource_group_name, cluster_name, nodepool_name, ignore_pod_disruption_budget=ignore_pod_disruption_budget)
+    else:
+        return sdk_no_wait(no_wait, client.begin_delete, resource_group_name, cluster_name, nodepool_name)
```

Comment:
Can we get rid of this `if ignore_pod_disruption_budget is True` judgment? When the value of `ignore_pod_disruption_budget` is `False`, the `ignore_pod_disruption_budget` parameter will not be passed in, dose this meet expectations?
```suggestion
   return sdk_no_wait(no_wait, client.begin_delete, resource_group_name, cluster_name, nodepool_name, ignore_pod_disruption_budget=ignore_pod_disruption_budget)
```

Reply:
This parameter is an optional one. When it is none or false, it doesn't take any effect. It makes no sense to input that parameter when it is false in this case.

Reply:
Yes, but I think so to make the code more concise. Other similar codes in the CLI do not have such `if-else` logic